lang/cs/Org.Apache.REEF.Network/packages.config

```diff
@@ -18,10 +18,49 @@ specific language governing permissions and limitations
 under the License.
 -->
 <packages>
+  <package id="Microsoft.Extensions.Caching.Abstractions" version="1.1.0" targetFramework="net451" />
+  <package id="Microsoft.Extensions.Caching.Memory" version="1.1.0" targetFramework="net451" />
+  <package id="Microsoft.Extensions.DependencyInjection.Abstractions" version="1.1.0" targetFramework="net451" />
+  <package id="Microsoft.Extensions.Options" version="1.1.0" targetFramework="net451" />
+  <package id="Microsoft.Extensions.Primitives" version="1.1.0" targetFramework="net451" />
```

Comment:
Why are all of these 4.5.1 packages being added. We need to maintain 4.5 compatibility?